packages/cashier/globals.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@deriv/account';
```

Comment:
@george-usynin-binary Please remove this file 🙏🏻

Reply:
@farzin-deriv if i remove this file i got an error when i try to import components from the account package
<img width="1035" alt="Screenshot 2022-11-24 at 15 13 57" src="https://user-images.githubusercontent.com/103181646/203781958-6342d476-f0f5-4457-aa02-be962733311e.png">

Reply:
we have this workaround in other packages, maybe there is a better solution to remove the error?

Reply:
@george-usynin-binary It's okay to have the error, We will gradually fix the errors in packages as we are migrating to TS.